plan/main

```diff
@@ -1,24 +1,28 @@
-#include <turnkey/base>
-#include <turnkey/lamp73>
-
-python3-bcrypt
-
-/* install specific versions as using non-standard PHP */
-php7.3-common
-php7.3-curl
-php7.3-json
-php7.3-mbstring
-php7.3-gd
-php7.3-ldap
-php7.3-zip
-php7.3-bcmath
-php7.3-xml
-php7.3-imap
-php7.3-redis
-
-redis-server
-supervisor
-
-unzip
-
-expect      /* Used to interact with interactive php aritsan commands */
+#include <turnkey/base>
+#include <turnkey/mysql>
+#include <turnkey/lamp81>
+
+php8.1-mysql
+libapache2-mod-php8.1
+php8.1-cli
+
+python3-argon2
```

Comment:
If this app is compatible with argon2 that's fine, but the [python inithook](https://github.com/turnkeylinux-apps/faveo-helpdesk/blob/master/overlay/usr/lib/inithooks/bin/faveo-helpdesk.py#L11) is still importing the bcrypt module (which is no longer being installed).

Having said that Faveo, still seems to work with bcrypt - so I don't think this change is necessary.

For context, Argon2 is a better (more secure) password hashing mechanism than Bcrypt (which IIRC is actually blowfish), but it will only work if the software (Faveo in this case) supports it.